postgres/patches/20240105_model-cursor-metadata-table.sql

```diff
@@ -0,0 +1,11 @@
+\connect dstk;
+
+CREATE SCHEMA metadata;
```

Comment:
The `metadata` schema should already exist and can just be reused without this `CREATE`

Reply:
I didn't see a `CREATE SCHEMA metadata` statement in any of the patches, or am I just stupid?

Reply:
Nevermind I am stupid, it's in the bootstrap query.